src/main/scala/is/hail/expr/ir/Compile.scala

```diff
@@ -0,0 +1,35 @@
+package is.hail.expr.ir
+
+import is.hail.annotations.Region
+import is.hail.asm4s.{AsmFunction5, FunctionBuilder, GenericTypeInfo, TypeInfo}
+import is.hail.expr.{HailRep, Type, ir}
```

Comment:
In the `ir` package, I've been generally avoiding prefixing things from `ir` with their package name (below `Infer` and `Emit` are). Just an aesthetic opinion on my part.

Reply:
Good, agreed.  I think this was just from a package move that didn't get cleaned up.